app/views/templates/_form.html.erb

```diff
@@ -1,40 +1,66 @@
-        <%= f.text_field :title, :class => 'form-control', :placeholder => 'Template name' %>
+        <%= f.text_field :title, class: 'form-control', placeholder: 'Template name' %>
     </div>
 
     <div class="form-group">
         <label>Description</label>
-        <%= f.text_area :description, :class=>"form-control rich-text-edit", rows: 5, id: "template-description" -%>
+        <%= f.text_area :description, class: "form-control rich-text-edit", rows: 5, id: "template-description" -%>
     </div>
 
-    <%= render :partial => "projects/project_selector", :locals => { :resource => @template } -%>
+    <%= render partial: "projects/project_selector", locals: { resource: @template } -%>
 
     <%= render partial: 'assets/manage_specific_attributes', locals:{f:f} if show_form_manage_specific_attributes? %>
 
-    <%= f.hidden_field :parent_id %>
-
     <div class="tab-content">
-        <div role="tabpanel" class="tab-pane<%= ' active'.html_safe if tab == 'manual' -%>" id="manual">
-            
-            <%= render :partial => "templates/template_modal" -%>
+        <div role="tabpanel" class="active" id="manual">
+
+            <%= render partial: "templates/template_modal" -%>
+            <div class="form-group">
+
+                <h3>Template Information</h3>
+
+                <% if @template.new_record? %>
+                    <div class="form-group">
+                        <input type="button" value="Choose from existing templates" class="btn btn-primary" onClick="showTemplateModal()">
+                    </div>
+                <% end %>
+
+                <div class="form-group">
+                    <%= f.label :level, 'ISA Level' %>
+                    <%= f.text_field :level, { class: 'form-control', readonly: true } %>
+                </div>
+                <div class="form-group">
+                    <%= f.label :organism %>
+                    <%= f.text_field :organism, {class: 'form-control'} %>
+                </div>
+                <div class="form-group">
+                    <%= f.label :version %>
+                    <%= f.text_field :version, {class: 'form-control', value: (@template.new_record? ? '1.0.0' : nil)} %>
+                </div>
+                <div class="form-group">
+                    <%= f.label :group %>
+                    <%= f.text_field :group, {class: 'form-control', readonly: true} %>
+                </div>
+                <div class="form-group">
+                    <%= f.hidden_field :parent_id %>
+                </div>
+
+            </div>
 
             <h3>Attributes</h3>
+            <div class="alert alert-warning alert-dismissible" role="alert">
+                <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+                <b class="text-danger">To ensure compliance with the original template, please do not modify the existing ISA-tags and attributes in the form below. You can always add new attribute.</b>
```

Comment:
Typo. Should read either "You can always add new attributes." or "You can always add a new attribute."